WordPress/src/main/java/org/wordpress/android/ui/plans/PlansUtils.java

```diff
@@ -201,6 +201,7 @@ public static boolean downloadGlobalPlans() {
             @Override
             public void onResponse(JSONObject response) {
                 if (response != null) {
+                    // TODO: don't log response - only here for testing
```

Comment:
> Is there a reason why we should not log the response?

I was just assuming it was for debugging - if you think that'll be helpful we can leave it in.